### ISSUE TITLE
Internal Error - Upload Image to a Profile

### ISSUE THREAD

**Issue by leomazzo** [passbolt/passbolt_api#64]
When we are trying to change the Photo (image) for a user profile, we have an error:

2017-04-21 21:38:04 Error: Fatal Error (64): Cannot use 'String' as class name as it is reserved in [/var/www/html/passbolt/lib/Cake/Utility/String.php, line 26]
2017-04-21 21:38:04 Error: [InternalErrorException] Internal Server Error
Request URL: /users/avatar/58fa9768-e98c-41ad-ad4f-61590a7b01bb
Stack Trace:
#0 /var/www/html/passbolt/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(64, 'Cannot use 'Str...', '/var/www/html/p...', 26)
#1 /var/www/html/passbolt/lib/Cake/Core/App.php(970): ErrorHandler::handleError(64, 'Cannot use 'Str...', '/var/www/html/p...', 26, Array)
#2 /var/www/html/passbolt/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}

We have the correct permissions on directory /var/www/html/passbolt/app/webroot/img/public

**Comment by Ducatel** [comment]
I have exactly the same error 
```
ll app/webroot/img/public
total 12
drwxrwxrwx 3 www-data www-data 4096 mai    3 15:15 ./
drwxr-xr-x 8 root     root     4096 mai    3 10:53 ../
-rwxrwxrwx 1 root     root        0 mai    3 10:53 empty*
drwxr-xr-x 3 www-data www-data 4096 mai    3 15:15 images/

```

**Comment by stripthis** [comment]
This looks like a php7 compatibility issue with the image library @kevinmuller .

**Comment by cedricalfonsi** [comment]
Fixed with v1.5.1
